modules/shBidAdapter.md

```diff
@@ -49,6 +49,42 @@ Module that connects to ShowHeroes demand source to fetch bids.
                    }
                ]
            },
+           {
+               code: 'video',
+               mediaTypes: {
+                   video: {
+                       playerSize: [640, 480],
+                       context: 'outstream',
+                   }
+               },
+               bids: [
+                   {
+                       bidder: "showheroes-bs",
+                       params: {
+                           playerId: '0151f985-fb1a-4f37-bb26-cfc62e43ec05',
+                           vpaidMode: true, // by default is 'false'
+                           outstreamOptions: { 
```

Comment:
Maybe add a note here that this is optional

Reply:
@muuki88 right, thanks! Will do :)

Reply:
Awesome :cocktail: